src/views/pages/Liquidity/Components/Charts/ChartVolume.js

```diff
@@ -37,7 +40,7 @@ const ChartVolume = (props) => {
     plugins: {
       title: {
         display: true,
-        text: 'Swap Volume in $USD',
+        text: t('swapVolumeInUsd'),
```

Comment:
Really appreciate you doing the legwork here! I need to get better at just injecting straight away instead of reactively